### PR TITLE
resize: adding window resize for cameras

### PR DESCRIPTION
**Pull request by munoz0raul**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-
+import cv2
```
Comment (mmphego) [inline main.py]: `cv2` forms part of `pyvino_utils` so we do not need to import it here.

https://github.com/B0N0AI/pyvino_utils/blob/master/pyvino_utils/input_handler/input_feeder.py#L4
```suggestion
```

File: main.py [hunk 2]
```diff
@@ -77,6 +77,18 @@ def arg_parser():
...
+        type=int,
+        default=640,
+        help="Input Width (Default: 640)",
+    )
+    parser.add_argument(
+        "--height",
+        type=int,
+        default=360,
+        help="Input Height (Default: 360)",
+    )
```
Comment (mmphego) [inline main.py:91]: :+1:

File: main.py [hunk 3]
```diff
@@ -92,6 +104,10 @@ def main(args):
     """
     # Initialise the video stream
     input_feed = InputFeeder(input_feed=args.input)
+    print('width = ' + str(args.width))
+    print('height = ' + str(args.height))
+    input_feed.cap.set(cv2.CAP_PROP_FRAME_WIDTH, args.width)
+    input_feed.cap.set(cv2.CAP_PROP_FRAME_HEIGHT, args.height)
```
Comment (mmphego) [inline main.py]: Perhaps, use the method that forms part of the `InputFeeder` class.
https://github.com/B0N0AI/pyvino_utils/blob/master/pyvino_utils/input_handler/input_feeder.py#L114+L119
```suggestion
input_feed.resize_cam_input(args.height, args.width)
```

**Review by mmphego — CHANGES_REQUESTED** [review]
Thanks for the change req.
Just minor recommendations and we are good to go.

**Comment by munoz0raul** [comment]
Hi @mmphego, I updated the commit with all changes you request. 

Raul.

**Comment by mmphego** [comment]
@munoz0raul Did you have time to test these changes?
I will merge this PR.

**Pull request merged**